src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -7,12 +7,32 @@
       size: 8.5in 11in;
       margin: 10%;
 
+      @bottom-center {
+        content: "For further details of the courts www.gov.uk/find-court-tribunal";
+        font-size: 10px;
+      }
+
+      @bottom-right {
+        content: "© Crown copyright Page " counter(page) " of " counter(pages);
+        /*noinspection CssNoGenericFontName*/
+        font-size: 10px;
+      }
+
       @bottom-left {
-        content: "OCON205A Request for judgment by default (07.17)";
-        font-size: 11px;
+        content: "OCON225";
+        /*noinspection CssNoGenericFontName*/
+        font-size: 10px;
       }
     }
 
+    /*@page :first {*/
+      /*@top-right {*/
+        /*content: "{{ ccj.claim.referenceNumber }}";*/
+        /*!*noinspection CssNoGenericFontName*!*/
+        /*font-family: "Open Sans";*/
+      /*}*/
+    /*}*/
+
```

Comment:
I need help here please - I want to get claim number only on second page as a header...I tried as shown above but doesn't work

Reply:
i got it to work now